### ISSUE TITLE
Feature Request: Minimum number of words

### ISSUE THREAD

**Issue by aksperiod** [mogelbrod/widont#3]
Typically I don't widont for huge titles with 3 or 4 words in them. It would be nice to add this feature.

**Comment by mogelbrod** [comment]
Is your goal to avoid widows when the string is 0-2 words and 5+ words, or only when it's 5+ words?

I recommend defining a custom helper function tailored to your use case. Here's two (untested) versions that might work for you:

```js
import widont from 'widont'

export function wimaybe1(input, minWords = 5, maxWords = '') {
  const regex = new RegExp('\S+(?:\s+\S+){' + minWords + ',' + maxWords + '}', 'g')
  return regex.test(input) ? widont(input) : input
}

export function wimaybe2(input) {
  const words = input.trim().split(/\s+/)
  return words < 3 || words > 4 ? widont(input) : input
}
```

**Comment by aksperiod** [comment]
only 5+ words, or whatever the minimum is. I know it’s an easy helper function, but so is widont’ing in general (I usually make my own function that adds `&nbsp;`, but I just discovered this library). Just thought it would be easy to add to the API and require minimal code added, with defaults such that there are no breaking changes.

**Comment by jayliu50** [comment]
Here to second this feature request!

**Comment by mogelbrod** [comment]
I released [v0.4.0](https://github.com/mogelbrod/widont/releases/tag/v0.4.0) which adds an optional argument to specify minimum number of words now. Please re-open this issue or create a new one if you encounter any problems with it!
